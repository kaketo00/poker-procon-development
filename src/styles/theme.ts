import { extendTheme } from '@chakra-ui/react';
import { Zen_Kaku_Gothic_New } from 'next/font/google';

const nextFont = Zen_Kaku_Gothic_New({
  weight: ['400', '700', '900'],
  subsets: ['latin'],
});

const theme = extendTheme({
  fonts: {
    heading: nextFont.style.fontFamily,
    body: nextFont.style.fontFamily,
  },
  colors: {
    brand100: {
      100: '#0DD203', // green 100%
      200: '#005CFF', // blue 100%
      300: '#B4171E', // red 100%
      600: '#777777', // dark-gray 100%
      700: '#D4D4D4', // gray 100%
      800: '#ECECEC', // light-gray 100%
    },
    brand80: {
      100: 'rgba(13, 210, 3, 0.8)', // green 80%
      200: 'rgba(0, 92, 255, 0.8)', // blue 80%
      300: 'rgba(180, 23, 30, 0.8)', // red 80%
      400: 'rgba(255, 255, 255, 0.8)', // white 80%
      500: 'rgba(0, 0, 0, 0.8)', // black 80%
      600: 'rgba(119, 119, 119, 0.8)', // gray 80%
    },
  },
  textStyles: {
    normal: {
      xs: {
        fontSize: '0.625rem',
        fontWeight: 'normal',
        lineHeight: '0.75rem',
      },
      sm: {
        fontSize: '0.75rem',
        fontWeight: 'normal',
        lineHeight: '1rem',
      },
      md: {
        fontSize: '1rem',
        fontWeight: 'normal',
        lineHeight: '1.25rem',
      },
      lg: {
        fontSize: '1.25rem',
        fontWeight: 'normal',
        lineHeight: '1.5rem',
      },
      xl: {
        fontSize: '1.5rem',
        fontWeight: 'normal',
        lineHeight: '1.75rem',
      },
    },
    bold: {
      xs: {
        fontSize: '0.625rem',
        fontWeight: 'bold',
        lineHeight: '0.75rem',
      },
      sm: {
        fontSize: '0.75rem',
        fontWeight: 'bold',
        lineHeight: '1rem',
      },
      md: {
        fontSize: '1rem',
        fontWeight: 'bold',
        lineHeight: '1.25rem',
      },
      lg: {
        fontSize: '1.25rem',
        fontWeight: 'bold',
        lineHeight: '1.5rem',
      },
      xl: {
        fontSize: '1.5rem',
        fontWeight: 'bold',
        lineHeight: '1.75rem',
      },
    },
    strong: {
      xs: {
        fontSize: '0.6875rem',
        fontWeight: 'black',
        lineHeight: '1rem',
      },
      sm: {
        fontSize: '1.25rem',
        fontWeight: 'black',
        lineHeight: '1.5rem',
      },
      md: {
        fontSize: '1.5rem',
        fontWeight: 'black',
        lineHeight: '2rem',
      },
      lg: {
        fontSize: '1.875rem',
        fontWeight: 'black',
        lineHeight: '2.375rem',
      },
      xl: {
        fontSize: '2.25rem',
        fontWeight: 'black',
        lineHeight: '2.75rem',
      },
    },
    heading: {
      h6: {
        fontSize: '1rem',
        fontWeight: 'black',
        lineHeight: '1.25rem',
      },
      h5: {
        fontSize: '1.25rem',
        fontWeight: 'black',
        lineHeight: '1.5rem',
      },
      h4: {
        fontSize: '1.5rem',
        fontWeight: 'black',
        lineHeight: '1.875rem',
      },
      h3: {
        fontSize: '1.75rem',
        fontWeight: 'black',
        lineHeight: '2.125rem',
      },
      h2: {
        fontSize: '2.25rem',
        fontWeight: 'black',
        lineHeight: '2.75rem',
      },
      h1: {
        fontSize: '1.875rem',
        fontWeight: 'black',
        lineHeight: '2.25rem',
      },
    },
    shadows: {
      light5: {
        100: '0px 0px 5px rgba(13, 210, 3, 0.8)',
        200: '0px 0px 5px rgba(0, 92, 255, 0.8)',
        300: '0px 0px 5px rgba(180, 23, 30, 0.8)',
        400: '0px 0px 5px rgba(255, 255, 255, 0.8)',
      },
      light10: {
        100: '0px 0px 10px rgba(13, 210, 3, 0.8)',
        200: '0px 0px 10px rgba(0, 92, 255, 0.8)',
        300: '0px 0px 10px rgba(180, 23, 30, 0.8)',
        400: '0px 0px 10px rgba(255, 255, 255, 0.8)',
      },
    },
  },
  breakpoints: {
    base: '0px', // 0 - 1279px game-template(S)
    sm: '80em', // 1280px - 1439px game-template(S)
    md: '90em', // 1440px - 1919px game-template(M)
    lg: '120em', // 1920px - 2559px game-template(M)
    xl: '160em', // 2560px - âˆž game-template(L)
    '2xl': '160em',
  },
});

export default theme;
