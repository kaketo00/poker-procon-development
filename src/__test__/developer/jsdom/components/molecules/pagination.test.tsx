import { ChakraProvider } from '@chakra-ui/react';
import { render, waitFor } from '@testing-library/react';
import userEvent from '@testing-library/user-event';

import type { PaginationProps } from '@/components/molecules/pagination';
import Pagination from '@/components/molecules/pagination';
import theme from '@/styles/theme';

const Component = (props: PaginationProps) => (
  <ChakraProvider theme={theme}>
    <Pagination {...props} />
  </ChakraProvider>
);

describe('Pagination', () => {
  test('スナップショット', () => {
    const { asFragment, getByText } = render(
      <Component page={1} max={1000} onClick={jest.fn()} />
    );

    expect(getByText('1,000件中 1 - 100件目を表示中')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });

  test('スナップショット/5ページ以下', () => {
    const { asFragment, getByText } = render(
      <Component page={1} max={300} onClick={jest.fn()} />
    );

    expect(getByText('300件中 1 - 100件目を表示中')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });

  test('スナップショット/最終3ページ以内', () => {
    const { asFragment, getByText } = render(
      <Component page={9} max={1000} onClick={jest.fn()} />
    );

    expect(getByText('1,000件中 801 - 900件目を表示中')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });

  test('スナップショット/最終ページが100件以下', () => {
    const { asFragment, getByText } = render(
      <Component page={10} max={999} onClick={jest.fn()} />
    );

    expect(getByText('999件中 901 - 999件目を表示中')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });

  test('ページ変更 - to first', async () => {
    const onClick = jest.fn();
    const { getByRole } = render(
      <Component page={5} max={1000} onClick={onClick} />
    );

    const next = getByRole('button', { name: 'first' });
    await userEvent.click(next);

    await waitFor(() => {
      expect(onClick).toHaveBeenCalledWith(1);
    });
  });

  test('ページ変更 - to prev', async () => {
    const onClick = jest.fn();
    const { getByRole } = render(
      <Component page={5} max={1000} onClick={onClick} />
    );

    const next = getByRole('button', { name: 'prev' });
    await userEvent.click(next);

    await waitFor(() => {
      expect(onClick).toHaveBeenCalledWith(4);
    });
  });

  test('ページ変更 - to number', async () => {
    const onClick = jest.fn();
    const { getByRole } = render(
      <Component page={5} max={1000} onClick={onClick} />
    );

    const next = getByRole('button', { name: 'number-6' });
    await userEvent.click(next);

    await waitFor(() => {
      expect(onClick).toHaveBeenCalledWith(6);
    });
  });

  test('ページ変更 - to next', async () => {
    const onClick = jest.fn();
    const { getByRole } = render(
      <Component page={5} max={1000} onClick={onClick} />
    );

    const next = getByRole('button', { name: 'next' });
    await userEvent.click(next);

    await waitFor(() => {
      expect(onClick).toHaveBeenCalledWith(6);
    });
  });

  test('ページ変更 - to last', async () => {
    const onClick = jest.fn();
    const { getByRole } = render(
      <Component page={5} max={1000} onClick={onClick} />
    );

    const next = getByRole('button', { name: 'last' });
    await userEvent.click(next);

    await waitFor(() => {
      expect(onClick).toHaveBeenCalledWith(10);
    });
  });
});
