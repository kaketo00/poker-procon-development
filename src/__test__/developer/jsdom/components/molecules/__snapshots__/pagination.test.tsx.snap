// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination スナップショット 1`] = `
<DocumentFragment>
  <div
    class="css-0"
  >
    <div
      class="css-1k9efnl"
    >
      <button
        aria-label="first"
        class="chakra-button css-mudq2d"
        disabled=""
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-129ongp"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M8.354 1.646a.5.5 0 0 1 0 .708L2.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill-rule="evenodd"
          />
          <path
            d="M12.354 1.646a.5.5 0 0 1 0 .708L6.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill-rule="evenodd"
          />
        </svg>
      </button>
      <button
        aria-label="prev"
        class="chakra-button css-mudq2d"
        disabled=""
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-onkibi"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M11.354 1.646a.5.5 0 0 1 0 .708L5.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill="currentColor"
          />
        </svg>
      </button>
      <button
        aria-label="number-1"
        class="chakra-button css-1ol6zar"
        type="button"
      >
        1
      </button>
      <button
        aria-label="number-2"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        2
      </button>
      <button
        aria-label="number-3"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        3
      </button>
      <button
        aria-label="number-4"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        4
      </button>
      <button
        aria-label="number-5"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        5
      </button>
      <svg
        class="chakra-icon css-u3gylf"
        focusable="false"
        viewBox="0 0 16 16"
      >
        <path
          d="M3 9.5a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3z"
          fill="currentColor"
        />
      </svg>
      <button
        aria-label="next"
        class="chakra-button css-mudq2d"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-onkibi"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z"
            fill="currentColor"
          />
        </svg>
      </button>
      <button
        aria-label="last"
        class="chakra-button css-mudq2d"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-u3gylf"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M3.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L9.293 8 3.646 2.354a.5.5 0 0 1 0-.708z"
            fill-rule="evenodd"
          />
          <path
            d="M7.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L13.293 8 7.646 2.354a.5.5 0 0 1 0-.708z"
            fill-rule="evenodd"
          />
        </svg>
      </button>
    </div>
    <p
      class="chakra-text css-vchh5a"
    >
      1,000件中 1 - 100件目を表示中
    </p>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;

exports[`Pagination スナップショット/5ページ以下 1`] = `
<DocumentFragment>
  <div
    class="css-0"
  >
    <div
      class="css-1k9efnl"
    >
      <button
        aria-label="first"
        class="chakra-button css-mudq2d"
        disabled=""
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-129ongp"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M8.354 1.646a.5.5 0 0 1 0 .708L2.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill-rule="evenodd"
          />
          <path
            d="M12.354 1.646a.5.5 0 0 1 0 .708L6.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill-rule="evenodd"
          />
        </svg>
      </button>
      <button
        aria-label="prev"
        class="chakra-button css-mudq2d"
        disabled=""
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-onkibi"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M11.354 1.646a.5.5 0 0 1 0 .708L5.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill="currentColor"
          />
        </svg>
      </button>
      <button
        aria-label="number-1"
        class="chakra-button css-1ol6zar"
        type="button"
      >
        1
      </button>
      <button
        aria-label="number-2"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        2
      </button>
      <button
        aria-label="number-3"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        3
      </button>
      <button
        aria-label="next"
        class="chakra-button css-mudq2d"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-onkibi"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z"
            fill="currentColor"
          />
        </svg>
      </button>
      <button
        aria-label="last"
        class="chakra-button css-mudq2d"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-u3gylf"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M3.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L9.293 8 3.646 2.354a.5.5 0 0 1 0-.708z"
            fill-rule="evenodd"
          />
          <path
            d="M7.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L13.293 8 7.646 2.354a.5.5 0 0 1 0-.708z"
            fill-rule="evenodd"
          />
        </svg>
      </button>
    </div>
    <p
      class="chakra-text css-vchh5a"
    >
      300件中 1 - 100件目を表示中
    </p>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;

exports[`Pagination スナップショット/最終3ページ以内 1`] = `
<DocumentFragment>
  <div
    class="css-0"
  >
    <div
      class="css-1k9efnl"
    >
      <button
        aria-label="first"
        class="chakra-button css-mudq2d"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-u3gylf"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M8.354 1.646a.5.5 0 0 1 0 .708L2.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill-rule="evenodd"
          />
          <path
            d="M12.354 1.646a.5.5 0 0 1 0 .708L6.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill-rule="evenodd"
          />
        </svg>
      </button>
      <button
        aria-label="prev"
        class="chakra-button css-mudq2d"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-onkibi"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M11.354 1.646a.5.5 0 0 1 0 .708L5.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill="currentColor"
          />
        </svg>
      </button>
      <svg
        class="chakra-icon css-u3gylf"
        focusable="false"
        viewBox="0 0 16 16"
      >
        <path
          d="M3 9.5a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3z"
          fill="currentColor"
        />
      </svg>
      <button
        aria-label="number-6"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        6
      </button>
      <button
        aria-label="number-7"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        7
      </button>
      <button
        aria-label="number-8"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        8
      </button>
      <button
        aria-label="number-9"
        class="chakra-button css-1ol6zar"
        type="button"
      >
        9
      </button>
      <button
        aria-label="number-10"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        10
      </button>
      <button
        aria-label="next"
        class="chakra-button css-mudq2d"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-onkibi"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z"
            fill="currentColor"
          />
        </svg>
      </button>
      <button
        aria-label="last"
        class="chakra-button css-mudq2d"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-u3gylf"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M3.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L9.293 8 3.646 2.354a.5.5 0 0 1 0-.708z"
            fill-rule="evenodd"
          />
          <path
            d="M7.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L13.293 8 7.646 2.354a.5.5 0 0 1 0-.708z"
            fill-rule="evenodd"
          />
        </svg>
      </button>
    </div>
    <p
      class="chakra-text css-vchh5a"
    >
      1,000件中 801 - 900件目を表示中
    </p>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;

exports[`Pagination スナップショット/最終ページが100件以下 1`] = `
<DocumentFragment>
  <div
    class="css-0"
  >
    <div
      class="css-1k9efnl"
    >
      <button
        aria-label="first"
        class="chakra-button css-mudq2d"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-u3gylf"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M8.354 1.646a.5.5 0 0 1 0 .708L2.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill-rule="evenodd"
          />
          <path
            d="M12.354 1.646a.5.5 0 0 1 0 .708L6.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill-rule="evenodd"
          />
        </svg>
      </button>
      <button
        aria-label="prev"
        class="chakra-button css-mudq2d"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-onkibi"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M11.354 1.646a.5.5 0 0 1 0 .708L5.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
            fill="currentColor"
          />
        </svg>
      </button>
      <svg
        class="chakra-icon css-u3gylf"
        focusable="false"
        viewBox="0 0 16 16"
      >
        <path
          d="M3 9.5a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3z"
          fill="currentColor"
        />
      </svg>
      <button
        aria-label="number-6"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        6
      </button>
      <button
        aria-label="number-7"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        7
      </button>
      <button
        aria-label="number-8"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        8
      </button>
      <button
        aria-label="number-9"
        class="chakra-button css-1dkcmnq"
        type="button"
      >
        9
      </button>
      <button
        aria-label="number-10"
        class="chakra-button css-1ol6zar"
        type="button"
      >
        10
      </button>
      <button
        aria-label="next"
        class="chakra-button css-mudq2d"
        disabled=""
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-onkibi"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z"
            fill="currentColor"
          />
        </svg>
      </button>
      <button
        aria-label="last"
        class="chakra-button css-mudq2d"
        disabled=""
        type="button"
      >
        <svg
          aria-hidden="true"
          class="chakra-icon css-129ongp"
          focusable="false"
          viewBox="0 0 16 16"
        >
          <path
            d="M3.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L9.293 8 3.646 2.354a.5.5 0 0 1 0-.708z"
            fill-rule="evenodd"
          />
          <path
            d="M7.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L13.293 8 7.646 2.354a.5.5 0 0 1 0-.708z"
            fill-rule="evenodd"
          />
        </svg>
      </button>
    </div>
    <p
      class="chakra-text css-vchh5a"
    >
      999件中 901 - 999件目を表示中
    </p>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;
